unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/SkyboxUtils.cs

```diff
@@ -43,60 +46,69 @@ public static class SkyboxShaderUtils
         public static readonly int FogIntensity = Shader.PropertyToID("_fogIntesity");
         public static readonly int Opacity = Shader.PropertyToID("_Opacity");
 
+        public const int TOTAL_SKYBOX_LAYERS = 5;
```

Comment:
could you move it at the top of the class, please. (just to be aligned with our code style)